tests/objects/test_operation.py

```diff
@@ -310,6 +313,7 @@ def test_writing_event_logs_to_disk(self, event_loop, op_for_event_logs, operati
             ),
             dict(
                 command='aG9zdG5hbWU=',
+                plaintext_command='aG9zdG5hbWU=',
```

Comment:
Seems like `plaintext_command` shouldn't be the same value as `command`, right?

Reply:
It seems that the normal `command` would get passed around as base64 (i.e. in the reports, API, etc), so the plaintext_command keeps the same paradigm. In these tests I think it's assumed that the command is unobfuscated from the operation, so the plaintext_command mirrors the command. If you run an operation with obfuscation, you'll find in the API and JSON reports that both are base64 encoded, but different.

I can definitely change it to be plaintext all the time, I just wasn't sure if that was a security issue to do so.